Zotero/Models/Files.swift

```diff
@@ -174,6 +174,12 @@ struct Files {
         return FileData(rootPath: Files.cachesRootPath, relativeComponents: ["Zotero", "shareext", "downloads"], name: "item_\(key)", contentType: contentType)
     }
 
+    // MARK: Epub/Html reader
+
+    static var tmpReaderDirectory: File {
+        return FileData.directory(rootPath: Files.cachesRootPath, relativeComponents: ["Zotero", UUID().uuidString])
```

Comment:
It seems that we use each tmp reader directory only once. If it's fast enough to create it, we can also delete it when the reader closes. Otherwise, maybe we can re-use it from the cache if it is still there?

Reply:
We already delete the directory in `HtmlEpubReaderActionHandler.deinitialise`. But I moved the call from `HtmlEpubDocumentViewController` to `HtmlEpubReaderViewController` which makes more sense. 450ed7e35c728f4f47d6c7199543ec5ad3528e63